content/guides/user-guides/delfin/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Delfin User Guides
```

Comment:
I think we need not add "User Guides" as it is already under User Guides and similar to what we followed for other items

Reply:
This is a page title. The menu title is just Delfin as other items.

Reply:
Looks like this in preview link

Reply:
Fixed

Reply:
@sushanthakumar you can check it out [here](https://deploy-preview-130--soda-docs.netlify.app/guides/user-guides/)